index.js

```diff
@@ -89,21 +90,22 @@ function run() {
     configPath: opts.gulpfile,
     require: opts.require,
     completion: opts.completion,
-  }, handleArguments);
+  }, configureAndInvoke);
 }
 
 module.exports = run;
 
-// The actual logic
-function handleArguments(env) {
+function configureAndInvoke(env) {
```

Comment:
Is this just split out due to the linting complaining about number of statements?  I'd be happier increasing the limit and putting it inside `handleArguments`.